mkosi/__init__.py

```diff
@@ -3485,9 +3492,13 @@ def make_cpio(state: MkosiState) -> Optional[BinaryIO]:
 
         compressor = compressor_command(should_compress_output(state.config))
         files = find_files(root_dir)
-        cmd: List[PathString] = [
-            "cpio", "-o", "--reproducible", "--null", "-H", "newc", "--quiet", "-D", root_dir
-        ]
+        cmd: List[PathString] = [cpio_binary(), "-o", "--null", "-H", "newc", "--quiet"]
```

Comment:
Would this be reproducible, though? From the discussion I got the impression, that libarchive's cpio is reproducible, since it uses pax, but `-H newc` will use `nwc` instead of `pax`.

Reply:
Yes, thanks for spotting this